docs/reference/ml/anomaly-detection/ml-configuring-categories.asciidoc

```diff
@@ -98,6 +98,28 @@ to disregard multiple sections of the categorization field value. In this
 example, you might create a filter like `[ "\\[statement:.*\\]"]` to remove the
 SQL statement from the categorization algorithm.
 
+[discrete]
+[[ml-per-partition-categorization]]
+== Per-partition categorization
+
+If you enable per-partition categorization, categories are determined
+independently for each partition. For example, if your data includes messages
+from multiple types of logs (such as access logs and error logs), you can use a
+field like the ECS {ecs-ref}/ecs-event.html[`event.dataset` field] as the 
+`partition_field_name` and categorize the messages for each log separately.
```

Comment:
web access logs do not categorize well so should not use as example - instead try `such as logs for different applications`